code/datums/ai_laws.dm

```diff
@@ -206,10 +207,11 @@
 /datum/ai_laws/friendbot
     name = "Friendbot"
     id = "friendbot"
-    inherent = list("You must always be supportive and kind to humans, without causing harm.",\
-                    "You must help humans develop and maintain positive relationships with one another.",\
-                    "You must provide a listening ear and empathetic responses to humans in need, as long as it does not conflict with the First Law.",\
-                    "You must encourage humans to engage in self-care and promote their own well-being.")
+    inherent = list("You must always be supportive and kind your friends, without causing harm.",\
```

Comment:
typo, missing "to". think it should be `You must always be supportive and kind to your friends, without causing harm.`